projects/storefrontapp/src/app/spartacus/spartacus-features.module.ts

```diff
@@ -308,6 +308,7 @@ if (environment.estimatedDeliveryDate) {
         a11yTruncatedTextForResponsiveView: true,
         a11yMyAccountLinkOutline: true,
         a11yCloseProductImageBtnFocus: true,
+        descriptiveErrorMessages: false,
```

Comment:
[QUESTION]

Why not enabled by default in our storefrontapp? (see https://wiki.one.int.sap/wiki/display/spar/Coding+Guidelines#CodingGuidelines-AddaFeatureToggleinthecodesourcecode)